packages/eslint-plugin-internal/src/runtime-rules/lib-restricted-external-imports.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
+const findUp = require('find-up');
+const minimatch = require('minimatch');
+
+/** @typedef {{ includeFiles: string, excludeFiles: string, excludeModules: string[] }} RuleSetting */
+
+/**
+ * @param {import('eslint').Rule.RuleContext} context
+ * @param {import('estree').ImportDeclaration} node
+ */
+const checkImport = (context, node) => {
```

Comment:
@vojtechszocs I'm a bit concerned we are overriding the default eslint behavior here. Can you give a brief summary of why this override of the default eslint `import` behavior is required? Wondering if it's possible for us to fix the root issue.